priv/apps/itest/lib/account.ex

```diff
@@ -2,15 +2,19 @@ defmodule Itest.Account do
   @moduledoc """
     Maintaining used accounts state so that we're able to run tests multiple times.
   """
-  use GenServer
+
   alias Itest.Transactions.Currency
-  @accounts_path "../data/ganache/data/account_keys.txt"
-  @accounts_state '/tmp/used_accounts'
+  alias Itest.Transactions.Encoding
+  import Itest.Poller, only: [wait_on_receipt_confirmed: 2]
+
+  @retry_count 60
 
+  # checksummed!!!
```

Comment:
I don't follow what you mean by `checksummed!!!`

Reply:
ugh, yeah, sorted in the next commit!